imgproxy/templates/_helpers.tpl

```diff
@@ -75,3 +75,12 @@ https://docs.aws.amazon.com/eks/latest/userguide/specify-service-account-role.ht
         {{- end }}
     {{- end }}
 {{- end -}}
+
+{{/* Return name for persistent volume claim */}}
+{{- define "imgproxy.pvcName" -}}
+{{- if .Values.persistence.existingClaim }}
+{{- print .Values.persistence.existingClaim }}
+{{- else -}}
+{{- print .Values.persistence.name }}
+{{- end -}}
```

Comment:
IMO it's safer to start from the root `$` here. And then some `default` based refactoring:

```suggestion
{{- with $.Values.persistence -}}
{{- .existingClaim | default .name | print }}
{{- end -}}
```